definitions/v3/blutopia-api.yml

```diff
@@ -54,10 +54,19 @@ search:
   paths:
     # https://hdinnovations.github.io/UNIT3D-Community-Edition-Docs/api_endpoints.html
     # https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/master/app/Http/Controllers/API/TorrentController.php
-    - path: "/api/torrents/filter?api_token={{ .Config.apikey }}&name={{ if .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}{{ if .Query.TMDBID }}&tmdbId={{ .Query.TMDBID }}{{ else }}{{ end }}{{ if .Query.IMDBIDShort }}&imdbId={{ .Query.IMDBIDShort }}{{ else }}{{ end }}{{ if  .Query.TVDBID }}&tvdbId={{ .Query.TVDBID }}{{ else }}{{ end }}&sortField={{ .Config.sort }}&sortDirection={{ .Config.type }}&perPage=100&page=1{{ range .Categories }}&categories[]={{.}}{{end}}{{ if .Config.freeleech }}&free=1{{ else }}{{ end }}"
+    - path: "/api/torrents/filter?sortField={{ .Config.sort }}&sortDirection={{ .Config.type }}&perPage=100&page=1{{ range .Categories }}&categories[]={{.}}{{end}}{{ if .Config.freeleech }}&free=1{{ else }}{{ end }}"
       response:
         type: json
         attribute: attributes
+  inputs:
+  # if we have an id based search, add Season and Episode as query in name for UNIT3D < v6.  Else pass S/E Params for UNIT3D >= v6
+    api_token: "{{ .Config.apikey }}"
+    name: "{{ if or (.Query.TVDBID) (.Query.IMDBID) (.Query.TMDBID) }}{{ if .Query.Season }}S{{ .Query.Season }}{{else}}{{ end }}{{ if .Query.Ep }}E{{ .Query.Ep }}{{else}}{{ end }}{{else}}{{ .Keywords }}{{ end }}"
+    tmdbid: "{{ if .Query.TMDBID }}{{ .Query.TMDBID }}{{else}}{{ end }}"
+    tvdbid: "{{ if .Query.TVDBID }}{{ .Query.TVDBID }}{{else}}{{ end }}"
+    imdbid: "{{ if .Query.IMDBIDShort }}{{ .Query.IMDBIDShort }}{{else}}{{ end }}"
+    # Supported with UNIT3d v6 only season: "{{ if .Query.Season }}{{ .Query.Season }}{{else}}{{ end }}"
```

Comment:
`{{ if .Query.Season }}{{ .Query.Season }}` gives error `Unexpceted type for variable .Query.Season: System.Int32`

Just `{{ .Query.Season }}` gives error `Unable to cast object of type 'System.Int32' to type 'System.String'`

Reply:
This might have been one we fixed in the cardigann implementation on our side, let me do some quick tests

Reply:
Any luck?

Reply:
Sorry been focusing on Radarr release. I'll have a go tomorrow

Reply:
Yep `{{ if .Query.Season }}{{ .Query.Season }}` seems to work fine on our side

```
[Trace] Http: Req: 47 [GET] /api/v1/search?query=%7BSeason%3A01%7D&indexerIds=103&type=tvsearch (from ::1)
[Info] ReleaseSearchService: Searching indexer(s): [Blutopia (API) Testd] for Term: [] for Season / Episode:[S01], Offset: 0, Limit: 0, Categories: []
[Info] Cardigann: Adding request: https://blutopia.xyz/api/torrents/filter??sortField=created_at&sortDirection=desc&perPage=100&page=1&api_token={cleaned}&name=&tmdbid=&tvdbid=&imdbid=&season=1&ep=
[Debug] Cardigann: Got 100 releases
[Trace] EventAggregator: Publishing IndexerQueryEvent
[Trace] EventAggregator: IndexerQueryEvent -> HistoryService
[Trace] EventAggregator: IndexerQueryEvent <- HistoryService
[Debug] ReleaseSearchService: Total of 100 reports were found for Term: [] for Season / Episode:[S01], Offset: 0, Limit: 0, Categories: [] from 1 indexer(s)
```

Reply:
Ah, any idea what your fix was?

Reply:
Yea let me find it, we did some typing stuff a while back to help solve some of these.

Reply:
Does Episode work as well?

Reply:
Yea `.Query.Episode` and `.Query.Ep` are working fine, it's just `.Query.Season` that is being a PitA.